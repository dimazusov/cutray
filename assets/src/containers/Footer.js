import React from 'react';

export default () => {
    return (
        <footer className="footer">
            <div className="logo footer__logo">
                <img className="logo__image logo__image_min" src="/img/logo.png" />
            </div>

            <div className="year footer__year">
                {( new Date()).getFullYear()}
            </div>
        </footer>
    );
}