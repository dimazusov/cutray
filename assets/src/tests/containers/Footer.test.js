import React from 'react';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });
import { shallow, } from 'enzyme';

import Footer from "../../containers/Footer";

it('renders without crashing', () => {
    const app = shallow(<Footer />);

    expect(app).toMatchSnapshot();
});