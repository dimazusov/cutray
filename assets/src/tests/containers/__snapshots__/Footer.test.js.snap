// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <_default />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="logo footer__logo"
        >
          <img
            className="logo__image logo__image_min"
            src="/img/logo.png"
          />
        </div>,
        <div
          className="year footer__year"
        >
          2020
        </div>,
      ],
      "className": "footer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            className="logo__image logo__image_min"
            src="/img/logo.png"
          />,
          "className": "logo footer__logo",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "logo__image logo__image_min",
            "src": "/img/logo.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": 2020,
          "className": "year footer__year",
        },
        "ref": null,
        "rendered": 2020,
        "type": "div",
      },
    ],
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="logo footer__logo"
          >
            <img
              className="logo__image logo__image_min"
              src="/img/logo.png"
            />
          </div>,
          <div
            className="year footer__year"
          >
            2020
          </div>,
        ],
        "className": "footer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              className="logo__image logo__image_min"
              src="/img/logo.png"
            />,
            "className": "logo footer__logo",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "logo__image logo__image_min",
              "src": "/img/logo.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": 2020,
            "className": "year footer__year",
          },
          "ref": null,
          "rendered": 2020,
          "type": "div",
        },
      ],
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
